nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/AbstractAzureDataLakeStorageProcessor.java

```diff
@@ -60,7 +71,8 @@
             .sensitive(true).build();
 
     public static final PropertyDescriptor ACCOUNT_KEY = new PropertyDescriptor.Builder()
-            .name("storage-account-key").displayName("Storage Account Key")
+            .name(STORAGE_ACCOUNT_KEY_PROPERTY_DESCRIPTOR_NAME)
```

Comment:
The property descriptor definition seems to be the same as in `AzureStorageUtils`.
The same descriptors could be used here for the Account Key and SAS Token (maybe for Endpoint Override too).